tests/packages/importlib_editable/pkg/__init__.py

```diff
@@ -0,0 +1,48 @@
+# Don't let ruff sort imports in this file, we want to keep them with the comments as is
+# for clarity.
+# ruff: noqa: I001
+
+# Level one pure modules
+from . import pmod_a
+
+# Level one extension modules
+from . import emod_a
+
+# Level one subpackages
+from . import sub_a, sub_b
+
+# Level two pure modules
+from .sub_a import pmod_b
+from .sub_b import pmod_c
+
+# Level two extension modules
+from .sub_a import emod_b
+from .sub_b import emod_c
+
+# Level two subpackages
+from .sub_b import sub_c, sub_d
```

Comment:
```suggestion
from .sub_b import sub_c
```

Let's focus the `import` on a single branch, and use the rest for navigation only.